ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -242,10 +243,10 @@ export const RuntimePanel = fp.flow(withCurrentWorkspace(), withStore(runtimeOps
         <FlexRow style={{justifyContent: 'flex-end', marginTop: '.75rem'}}>
           <Button disabled={true}>Create</Button>
         </FlexRow>
-        {outstandingRuntimeOp && <hr/>}
-        {outstandingRuntimeOp && <div>
-          <OutstandingRuntimeOp operation={outstandingRuntimeOp.operation} workspaceNamespace={workspace.namespace}/>
-        </div>}
+        {outstandingRuntimeOp && <React.Fragment>
+          <hr/>
+          <ActiveRuntimeOp operation={outstandingRuntimeOp.operation} workspaceNamespace={workspace.namespace}/>
```

Comment:
nit: I would either change all of the variable naming to go from outstanding -> active or only use the "Active" term on the user facing string.

Having both synonymous terms like in this line can be confusing when searching for code references.

Reply:
whoops, thought I got them all. Thank you!